docs/source/tutorial/02-Aggregates-Joins.ipynb

```diff
@@ -0,0 +1,844 @@
+       "4             AND        EU  1974  1.865587e+08\n",
+       "...           ...       ...   ...           ...\n",
+       "9482          ZWE        AF  2013  1.909102e+10\n",
+       "9483          ZWE        AF  2014  1.949552e+10\n",
+       "9484          ZWE        AF  2015  1.996312e+10\n",
+       "9485          ZWE        AF  2016  2.054868e+10\n",
+       "9486          ZWE        AF  2017  2.281301e+10\n",
+       "\n",
+       "[9487 rows x 4 columns]"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gdp_with_continent = countries_and_gdp[countries_simplified['country_code'],\n",
+    "                                       countries_simplified['continent'],\n",
+    "                                       gdp['year'],\n",
+    "                                       gdp['value'].name('gdp')].materialize()\n",
+    "gdp_with_continent"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "    country_code continent\n",
+       "0            AND        EU\n",
+       "1            ARE        AS\n",
+       "2            AFG        AS\n",
+       "3            ATG        NA\n",
+       "4            AIA        NA\n",
+       "..           ...       ...\n",
+       "247          YEM        AS\n",
+       "248          MYT        AF\n",
+       "249          ZAF        AF\n",
+       "250          ZMB        AF\n",
+       "251          ZWE        AF\n",
+       "\n",
+       "[252 rows x 2 columns]"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "countries_simplified"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "     country_code  year         value\n",
+       "0             ABW  1986  4.054634e+08\n",
+       "1             ABW  1987  4.876025e+08\n",
+       "2             ABW  1988  5.964236e+08\n",
+       "3             ABW  1989  6.953044e+08\n",
+       "4             ABW  1990  7.648871e+08\n",
+       "...           ...   ...           ...\n",
+       "9995          SVK  2002  3.513034e+10\n",
+       "9996          SVK  2003  4.681659e+10\n",
+       "9997          SVK  2004  5.733202e+10\n",
+       "9998          SVK  2005  6.278531e+10\n",
+       "9999          SVK  2006  7.070810e+10\n",
+       "\n",
+       "[10000 rows x 3 columns]"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "gdp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {},
+   "outputs": [
+    {
+     "ename": "IntegrityError",
+     "evalue": "Duplicate column name(s): ['country_code']",
```

Comment:
I'm getting this error as well, but it isn't explained so I'm guessing it's not supposed to be erroring?